stdlib/math/math.flux

```diff
@@ -51,154 +75,230 @@ builtin maxuint : uint
 // >     |> map(fn: (r) => ({r with _value: math.abs(x: r._value)}))
 // ```
 //
+// ## Special cases
+//
+// ```no_run
+// math.abs(x: ±Inf) // Returns +Inf
```

Comment:
The `±` is not valid Flux. I know there were examples like this in Flux comments before. I think it might be better to explicitly mark the cases for both `+Inf` and `-Inf` if needed for these examples and the others, sadly. Thoughts?

Reply:
Or I guess your treating the special cases not as literal Flux code? Looking over some of the other special cases (like where the special case is just one parameter and the other parameter can be anything) I can go either way now, I'll leave it up to your discretion.

Reply:
Yeah, I had this same thought. These special cases are actually pulled from the Go math package documentation. Some of the other special cases would be hard to accurately represent with fully valid Flux. For example,`math.acos(x: <-1)`. Just including a value that is less than -1 doesn't accurately reflect the scope of the special case.

For what it's worth, these special cases have been in the existing math package docs since it was released and I've never heard any complaints about them not working. Doesn't mean that users haven't tried, but I haven't heard anything.